src/main/kotlin/no/nav/tilleggsstonader/sak/vilkår/vilkårperiode/domain/Vilkårperiode.kt

```diff
@@ -109,11 +109,16 @@ enum class ResultatDelvilkårperiode {
 
 data class DelvilkårMålgruppe(
     val medlemskap: Vurdering,
+    val dekketAvAnnetRegelverk: Vurdering,
 ) : DelvilkårVilkårperiode() {
     init {
         brukerfeilHvis(medlemskap.resultat == ResultatDelvilkårperiode.IKKE_OPPFYLT && medlemskap.begrunnelse == null) {
             "Mangler begrunnelse for ikke oppfylt medlemskap"
         }
+
+        brukerfeilHvis(dekketAvAnnetRegelverk.resultat == ResultatDelvilkårperiode.IKKE_OPPFYLT && dekketAvAnnetRegelverk.begrunnelse == null) {
```

Comment:
Tenker det kan være fint å sjekke at begrunnelser/tekster fra forntend som ikke skal være null sjekker at de ikke er null eller blank med eks `.isNullOrBlank()`
```suggestion
        brukerfeilHvis(medlemskap.resultat == ResultatDelvilkårperiode.IKKE_OPPFYLT && medlemskap.begrunnelse.isNullOrBlank()) {
            "Mangler begrunnelse for ikke oppfylt medlemskap"
        }

        brukerfeilHvis(dekketAvAnnetRegelverk.resultat == ResultatDelvilkårperiode.IKKE_OPPFYLT && dekketAvAnnetRegelverk.begrunnelse.isNullOrBlank()) {
```